src/buildercore/cfngen.py

```diff
@@ -409,7 +409,7 @@ def _parameterize_bigquerylogging(bigquerylogging):
         return bigquerylogging
 
     context['fastly'] = False
-    if pdata['aws'].get('fastly'):
+    if pdata['aws'].get('fastly') and pdata['domain']:
```

Comment:
seems fair as `domain` is the top-level `elifesciences.org` used to build DNS entries; hence if it's not around it means no DNS entries are possible and hence no CDN can be setup.

I probably comment out `fastly` along with setting `"domain": False` when migrating infrastructure.

Reply:
I'll include your comment in the code